src/addons/send2ue/ui/dialog.py

```diff
@@ -63,6 +63,7 @@ def draw_export_tab(self, layout):
         properties = bpy.context.scene.send2ue
         self.draw_property(properties, layout, 'use_object_origin')
         self.draw_property(properties, layout, 'export_object_name_as_root')
+        self.draw_property(properties, layout, 'export_custom_root_name')
```

Comment:
I think it would be good if the user can see this setting are exclusive of each other by greying out `export_custom_root_name` if `export_object_name_as_root` is true. I believe you could do it like this, but havent tested.

`self.draw_property(properties, layout, 'export_custom_root_name', enabled=not properties.export_object_name_as_root)`

Reply:
I will approve review once that suggestion is implemented

Reply:
Hi @jack-yao91 , @JoshQuake . Apologies for not getting to the feedback in time, but thank you for sorting & merging this. Much appreciated.

Reply:
@vaeryn-uk All good! Thank YOU for your contribution!